### ISSUE TITLE
saliency + guided backprop + all filters + linear activation + VGG16 gives zero gradients

### ISSUE THREAD

**Issue by eyaler** [raghakot/keras-vis#200]
not sure why but the combination of guided backprop + filter_indices=None (all filters) + linear final activation gives me zero gradients for VGG16. you can see this in the demo notebook.

**Comment by eyaler** [comment]
i now also see this sometimes with softmax
